FSharp-7.0/FS-1124-interfaces-with-static-abstract-members.md

```diff
@@ -61,6 +61,8 @@ Static virtual methods can be called directly on interfaces.
 
 Static abstract methods may not be called directly on interfaces.
 
+Static abstract methods may not be declared on abstract classes.
+
```

Comment:
Maybe we should just update this sentence above

https://github.com/fsharp/fslang-design/blob/fb616a8136c28f1e3ba7be930c3b86168af0dbc0/FSharp-7.0/FS-1124-interfaces-with-static-abstract-members.md?plain=1#L58

to be

```diff
-Static abstract methods may not be declared in classes.
+Static abstract methods may not be declared in classes, abstract classes, or structs.
```

Reply:
Yeah, make sense.